java2/Makefile

```diff
@@ -1,5 +1,12 @@
 %.class: src/mal/
 	@rm -rf $*
+	@test -d bin || mkdir -p bin 
+	@if [ ! -f bin/jline-2.13.jar ]; \
+		then \
+		echo "downloading jline-2.13.jar form maven repo";\
+	  wget -O bin/jline-2.13.jar "http://central.maven.org/maven2/jline/jline/2.13/jline-2.13.jar" ;\
```

Comment:
Why use `wget` if you've installed `curl` in the Dockerfile?

Reply:
thanks,  it fixed.